.github/workflows/kvrocks.yaml

```diff
@@ -158,10 +156,13 @@ jobs:
         run: ./x.py build -j$NPROC --unittest --compiler ${{ matrix.compiler }} ${{ matrix.with_ninja }} ${{ matrix.with_sanitizer }} ${{ matrix.without_jemalloc }} ${{ matrix.without_luajit }}
 
       - name: Run Unit Test
-        run: ./build/unittest
+        run: |
+          export ${{ matrix.runtime_env_vars }}
+          ./build/unittest
 
       - name: Run Redis Tcl Test
         run: |
+          export ${{ matrix.runtime_env_vars }}
```

Comment:
You can use `env` field of a step.

See also https://docs.github.com/en/actions/learn-github-actions/environment-variables.

Reply:
I think it is hard to eval something in `env` like `$(realpath ./tests/tsan-suppressions)`, do you have any suggestions?

Reply:
Aha. I get your point. Then let's keep this logic as is.